lib/libc/Kconfig

```diff
@@ -24,6 +24,13 @@ config MINIMAL_LIBC
 	help
 	  Build with minimal C library.
 
+config PICOLIBC
+	bool "Picolibc library"
+	depends on !NATIVE_APPLICATION
+	help
+	  Build with picolibc library. The picolibc library is expected to be
+	  part of the SDK in this case.
```

Comment:
Sorry for using the wrong term; I was using 'SDK' in a more general way. But, yes, picolibc could be added to the Zephyr SDK as well so that users can get tooling which is known to work well. Perhaps 'expected to be already installed for use with the toolchain'.

Reply:
Maybe you could rephrase this help text ?